xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js

```diff
@@ -1028,7 +1028,8 @@ var LiveTablePagination = Class.create({
    * @return an Element containing the select
    **/
   createPageSizeSelectControl: function() {
-    var select = new Element('select', {'class':'pagesizeselect'});
+    var select = new Element('select', {'class':'pagesizeselect',
+    'aria-label':"$escapetool.javascript($services.localization.render('platform.livetable.selectPageSize.label'))"});
```

Comment:
The formatting looks wrong around ':'. Also, isn't the line too long?

Reply:
Swapped the quotation marks :+1: 
L1040 was still longer. I fixed both to make it more readable.